ansible/projects/ovn-kubernetes/environment.sh

```diff
@@ -0,0 +1,2 @@
+SWITCHDEV=true
```

Comment:
Should it be `export SWITCHDEV=true`?

Reply:
This will also work, since we do the export again in the run_kind_ci.sh when the variables are read from CLI.